apps/web/screens/Grants/SearchResults/SearchResults.tsx

```diff
@@ -0,0 +1,470 @@
+                          <PlazaCard
+                            eyebrow={grant.fund?.title ?? grant.name ?? ''}
+                            subEyebrow={grant.fund?.parentOrganization?.title}
+                            title={grant.name ?? ''}
+                            text={grant.description ?? ''}
+                            logo={
+                              grant.fund?.parentOrganization?.logo?.url ?? ''
+                            }
+                            logoAlt={
+                              grant.fund?.parentOrganization?.logo?.title ?? ''
+                            }
+                            tag={{
+                              label: grant.statusText ?? '',
+                              variant: tagVariant,
+                            }}
+                            cta={{
+                              label: 'Skoða nánar',
+                              variant: 'text',
+                              onClick: () => {
+                                router.push(
+                                  linkResolver(
+                                    'styrkjatorggrant',
+                                    [grant?.applicationId ?? ''],
+                                    locale,
+                                  ).href,
+                                )
+                              },
+                              icon: 'arrowForward',
+                            }}
+                            detailLines={[
+                              grant.dateFrom && grant.dateTo
+                                ? {
+                                    icon: 'calendar' as const,
+                                    text: `${format(
+                                      new Date(grant.dateFrom),
+                                      'dd.MM.',
+                                    )}-${format(
+                                      new Date(grant.dateTo),
+                                      'dd.MM.yyyy',
+                                    )}`,
+                                  }
+                                : null,
+                              {
+                                icon: 'time' as const,
+                                //todo: fix when the text is ready
+                                text: 'Frestur til 16.08.2024, kl. 15.00',
+                              },
+                              grant.categoryTag?.title
+                                ? {
+                                    icon: 'informationCircle' as const,
+                                    text: grant.categoryTag.title,
+                                  }
+                                : undefined,
+                            ].filter(isDefined)}
+                          />
+                        )}
+                      </Box>
+                    )
+                  })}
+                </Inline>
+              ) : undefined}
+              {!grants?.length && !error && !loading && (
```

Comment:
Can we also add `&& called &&` at the back so we don't see the "Nothing was found" at the start

Reply:
Good call